### ISSUE TITLE
Simulation page doesn't show any data on Edge browser

### ISSUE THREAD

**Issue by brantonb** [boknows/cFIREsim-open#58]
I'm running Windows 10 and using Microsoft Edge. The graphs on the simulation are just white boxes and there's nothing under Statistics.

**Comment by boknows** [comment]
Yeah. This is a bug in the graphing library that I use.  It was present in the original pre-open-source version of cFIREsim.  I had basically just said "screw IE browsers. Use a real browser."   I'll look into fixing this now.

**Comment by brantonb** [comment]
Note that it works in the version of IE in Windows 10.

**Comment by boknows** [comment]
Interesting. You're right.  It *shouldn't* work in IE at all (it didn't in
cFIREsim legacy).  I'll have to double check what is going on in Edge when
I get on my Win10 machine tonight. Thanks.

**Comment by tster123** [comment]
It works in the new chromium Edge.
